src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String STR_SUPPLY = "supply,";
```

Comment:
bad naming of constants. 
SUPPLY = "supply"; 
BUY = "buy";
Don't need to use comma in this constants, as you already have COMMA_SYMBOL because it gives flexibility to change value of constant when you need